<ui version="4.0" >
 <class>SMSD_Form</class>
 <widget class="QMainWindow" name="SMSD_Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QTabWidget" name="mainTabWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>170</y>
      <width>896</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>1</number>
    </property>
    <widget class="QWidget" name="basics_tab" >
     <attribute name="title" >
      <string>Basics</string>
     </attribute>
     <widget class="QSpinBox" name="massSpinBox" >
      <property name="geometry" >
       <rect>
        <x>70</x>
        <y>10</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="minimum" >
       <number>1</number>
      </property>
      <property name="maximum" >
       <number>9000000</number>
      </property>
      <property name="singleStep" >
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>51</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text" >
       <string>Volume</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text" >
       <string>Mass</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="volumeSpinBox" >
      <property name="geometry" >
       <rect>
        <x>70</x>
        <y>40</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="minimum" >
       <number>3</number>
      </property>
      <property name="maximum" >
       <number>27000000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4" >
      <property name="geometry" >
       <rect>
        <x>200</x>
        <y>17</y>
        <width>32</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text" >
       <string>Tons</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5" >
      <property name="geometry" >
       <rect>
        <x>200</x>
        <y>40</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text" >
       <string>Cumets</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>70</y>
        <width>341</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title" >
       <string>Class</string>
      </property>
      <widget class="QTableWidget" name="classTableWidget" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>321</width>
         <height>151</height>
        </rect>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text" >
         <string>Mass</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Mass Category</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Category #</string>
        </property>
       </column>
       <item row="0" column="0" >
        <property name="text" >
         <string>1-999</string>
        </property>
       </item>
       <item row="0" column="1" >
        <property name="text" >
         <string>Small</string>
        </property>
       </item>
       <item row="0" column="2" >
        <property name="text" >
         <string>1</string>
        </property>
       </item>
       <item row="1" column="0" >
        <property name="text" >
         <string>1K-99,999</string>
        </property>
       </item>
       <item row="1" column="1" >
        <property name="text" >
         <string>Medium</string>
        </property>
       </item>
       <item row="1" column="2" >
        <property name="text" >
         <string>2</string>
        </property>
       </item>
       <item row="2" column="0" >
        <property name="text" >
         <string>100K-999,999</string>
        </property>
       </item>
       <item row="2" column="1" >
        <property name="text" >
         <string>Large</string>
        </property>
       </item>
       <item row="2" column="2" >
        <property name="text" >
         <string>3</string>
        </property>
       </item>
       <item row="3" column="0" >
        <property name="text" >
         <string>1000K+</string>
        </property>
       </item>
       <item row="3" column="1" >
        <property name="text" >
         <string>Super Large</string>
        </property>
       </item>
       <item row="3" column="2" >
        <property name="text" >
         <string>4</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="geometry" >
       <rect>
        <x>360</x>
        <y>0</y>
        <width>521</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title" >
       <string>Hull</string>
      </property>
      <widget class="QTableWidget" name="hullTableWidget" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>50</y>
         <width>501</width>
         <height>431</height>
        </rect>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <row>
        <property name="text" >
         <string>21</string>
        </property>
       </row>
       <row>
        <property name="text" >
         <string>22</string>
        </property>
       </row>
       <row>
        <property name="text" >
         <string>23</string>
        </property>
       </row>
       <row>
        <property name="text" >
         <string>24</string>
        </property>
       </row>
       <column>
        <property name="text" >
         <string>CAT</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Description</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Volume Factor</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Volume</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Cost Multiplier</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Cost</string>
        </property>
       </column>
       <item row="0" column="0" >
        <property name="text" >
         <string>21</string>
        </property>
       </item>
       <item row="0" column="1" >
        <property name="text" >
         <string>Steel</string>
        </property>
       </item>
       <item row="0" column="2" >
        <property name="text" >
         <string>0.02</string>
        </property>
       </item>
       <item row="0" column="4" >
        <property name="text" >
         <string>1.0</string>
        </property>
       </item>
       <item row="1" column="0" >
        <property name="text" >
         <string>22</string>
        </property>
       </item>
       <item row="1" column="1" >
        <property name="text" >
         <string>Hardened Steel</string>
        </property>
       </item>
       <item row="1" column="2" >
        <property name="text" >
         <string>0.02</string>
        </property>
       </item>
       <item row="1" column="4" >
        <property name="text" >
         <string>1.5</string>
        </property>
       </item>
       <item row="2" column="0" >
        <property name="text" >
         <string>23</string>
        </property>
       </item>
       <item row="2" column="1" >
        <property name="text" >
         <string>Crysteel</string>
        </property>
       </item>
       <item row="2" column="2" >
        <property name="text" >
         <string>0.02</string>
        </property>
       </item>
       <item row="2" column="4" >
        <property name="text" >
         <string>3.0</string>
        </property>
       </item>
       <item row="3" column="0" >
        <property name="text" >
         <string>24</string>
        </property>
       </item>
       <item row="3" column="1" >
        <property name="text" >
         <string>Crystanium</string>
        </property>
       </item>
       <item row="3" column="2" >
        <property name="text" >
         <string>0.02</string>
        </property>
       </item>
       <item row="3" column="4" >
        <property name="text" >
         <string>5.5</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="alloyDBComboBox" >
       <property name="geometry" >
        <rect>
         <x>100</x>
         <y>20</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>DB +5 cost: 2.0</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>DB +10 cost: 4.0</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>DB +15 cost: 8.0</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>DB +20 cost: 16.0</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>DB +25 cost: 32.0</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_7" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text" >
        <string>Superior alloy</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8" >
       <property name="geometry" >
        <rect>
         <x>250</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text" >
        <string>Superior alloy</string>
       </property>
      </widget>
      <widget class="QComboBox" name="alloyVFComboBox" >
       <property name="geometry" >
        <rect>
         <x>340</x>
         <y>20</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>-10% Volume cost: 2.0</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>-20% Volume cost: 4.0</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>-40% Volume cost: 8.0</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>-60% Volume cost: 16.0</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>-80% Volume cost: 32.0</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>440</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title" >
       <string>Atmospheric Streamlining</string>
      </property>
      <widget class="QCheckBox" name="streamlinedCheckBox" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text" >
        <string>Streamlined</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="streamlinedCostLineEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="geometry" >
        <rect>
         <x>200</x>
         <y>20</y>
         <width>131</width>
         <height>26</height>
        </rect>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_15" >
       <property name="geometry" >
        <rect>
         <x>160</x>
         <y>20</y>
         <width>54</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text" >
        <string>Cost</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="drives_tab" >
     <attribute name="title" >
      <string>Drives</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>10</y>
        <width>351</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title" >
       <string>Sublight Drive</string>
      </property>
      <widget class="QTableWidget" name="tableWidget" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>331</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>390</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title" >
       <string>Power</string>
      </property>
      <widget class="QWidget" name="" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>202</width>
         <height>74</height>
        </rect>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_16" >
          <property name="text" >
           <string>Fuel Storage</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QSpinBox" name="fuelStorageSpinBox" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_11" >
          <property name="text" >
           <string>Volume</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="powerVolumeLineEdit" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>Rating * Category # * Category #</string>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_12" >
          <property name="text" >
           <string>Cost</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="powerCostLineEdit" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>Rating * Category # * Category # * 500 + 50K</string>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="systems_tab" >
     <attribute name="title" >
      <string>Systems</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title" >
       <string>Relative Inertial Field Generator</string>
      </property>
      <widget class="QLabel" name="label_9" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>50</y>
         <width>54</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text" >
        <string>Cost</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>54</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text" >
        <string>Volume</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="rifVolumeLineEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="geometry" >
        <rect>
         <x>60</x>
         <y>20</y>
         <width>131</width>
         <height>26</height>
        </rect>
       </property>
       <property name="toolTip" >
        <string>Ship Volume * 0.01</string>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="rifCostLineEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="geometry" >
        <rect>
         <x>60</x>
         <y>50</y>
         <width>131</width>
         <height>26</height>
        </rect>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>100</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title" >
       <string>Landing Gear</string>
      </property>
      <widget class="QLabel" name="label_13" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>80</y>
         <width>54</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text" >
        <string>Cost</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>50</y>
         <width>54</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text" >
        <string>Volume</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="landingGearVolumeLineEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="geometry" >
        <rect>
         <x>60</x>
         <y>50</y>
         <width>131</width>
         <height>26</height>
        </rect>
       </property>
       <property name="toolTip" >
        <string>Ship Volume * 0.01</string>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="landingGearCostLineEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="geometry" >
        <rect>
         <x>60</x>
         <y>80</y>
         <width>131</width>
         <height>26</height>
        </rect>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="landingGearCheckBox" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text" >
        <string>Landing Gear</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="armaments_tab" >
     <attribute name="title" >
      <string>Armaments</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="facilities_tab" >
     <attribute name="title" >
      <string>Facilities</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>896</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title" >
     <string>Ship</string>
    </property>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Available Volume</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="volumeProgressBar" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="value" >
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>110</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Cost</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="costLineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>40</x>
       <y>110</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>Cost for the ship</string>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="availableVolumeLabel" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>volume / volume</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
   <property name="shortcut" >
    <string>X</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
